source/codegen/metadata/nifake/functions.py

```diff
@@ -743,7 +743,7 @@
                 'name': 'numberOfElements',
                 'size': {
                     'mechanism': 'custom-code',
-                    'value': None
+                    'value': '',
```

Comment:
note: all of these get mocked out anyway, but we don't generally allow `None` as a custom-code value so I changed it here.